README.md

```diff
@@ -219,6 +219,34 @@ And add your labels to your translations to keep everything tidy.
 
 This example is a super-simple implementation. You can define your Gates as in any standard Laravel Application and can simply add the additional checks to validate them against your assigned Roles and Permissions.
 
+## Private Access
+
+Sometimes you might want to prevent some users from access content, but not others. To achieve this, use the included `IsAccessible.php` trait on your model.
+
+To check if a user has the correct permissions to view your content, add the following somewhere in your controller methods (Replacing 'Model' with your desired model.): 
+
+```php
+public function show(Model $model)
+{
+    // ...
+    abort_unless(Gate::allows('accessContent', $model->access), 403);
+    // ...
+}
+```
+
+On your Nova resource, add the AccessControl field:
+
+```php
+public function fields(Request $request)
+{
+    return [
+        // ...
+        AccessControl::make(),
+        // ...
+    ]
+}
+```
+
```

Comment:
Would probably be useful to document how this works within Nova too. E.g. Only roles which `canBeGivenPrivateAccess` will appear in the dropdown, and to protect content from being access, at least one Role has to be given access, otherwise the resource will be available to everyone.